src/context/UserLoggedContext.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { createContext, useState, useContext } from 'react';
```

Comment:
I learned a lot from this, I think adding global context allows our app to scale exponentially and seamlessly. Any way to avoid excessive prop drilling is a win in my book. Looks good, excited to implement.

Reply:
Exactly! Apologies for taking so long! We'll be implementing it in a bit of a divergent way but it makes it easy for implementation for the end dev in this case lol.